tests/build/test_build_disease.py

```diff
@@ -13,7 +13,7 @@ def test_build_disease_term(adapter, test_disease):
 
     ## THEN assert the term is on the correct format
 
-    assert disease_obj["_id"] == disease_obj["disease_id"] == "OMIM:615349"
```

Comment:
Nice, but there are still some more tests assuming `_id` is `disease_id`, especially for `omim_term`.

Reply:
There is also another in the cli:

```
diff --git a/scout/commands/view/diseases.py b/scout/commands/view/diseases.py
index 18ed4027d..f99ffb849 100644
--- a/scout/commands/view/diseases.py
+++ b/scout/commands/view/diseases.py
@@ -23,5 +23,5 @@ def diseases():
     else:
         click.echo("Disease")
         for disease_obj in adapter.disease_terms():
-            click.echo("{0}".format(disease_obj["_id"]))
+            click.echo("{0}".format(disease_obj["disease_id"]))
         LOG.info("{0} diseases found".format(nr_diseases))
```

Reply:
Thanks, I'll need to take a look at it again before it's going to be ready for review again. I also need to fix a couple of conflicts. Will do!